imagefudge/image_fudge.py

```diff
@@ -34,33 +34,41 @@ def draw_relative_arcs(self, origins, endpoints, arclen):
             provide support for true negative arclen
         """
 
+        try:
+            if arclen > 360 and arclen < -360:
+                raise ValueError('arclen must be between -360 and 360')
```

Comment:
We should get rid of ValueErrors and throw warnings instead since negative numbers and numbers > 360 don't cause failure